modules/openapi-generator/src/main/resources/Java/libraries/jersey2/anyof_model.mustache

```diff
@@ -48,6 +52,19 @@ public class {{classname}} extends AbstractOpenApiSchema{{#vendorExtensions.x-im
             {{/anyOf}}
             throw new IOException(String.format("Failed deserialization for {{classname}}: no match found"));
         }
+
+        /**
+         * Handle deserialization of the 'null' value.
+         */
+        @Override
+        public {{classname}} getNullValue(DeserializationContext ctxt) throws JsonMappingException {
+        {{#isNullable}}
+            return null;
+        {{/isNullable}}
+        {{^isNullable}}
+            throw new JsonMappingException("{{classname}} cannot be null");
```

Comment:
@wing328 , is there a better way to fail deserialization of the `null` value that would work across all OAS schema types?